activesupport/lib/active_support/string_inquirer.rb

```diff
@@ -8,6 +8,12 @@ module ActiveSupport
   # you can call this:
   #
   #   Rails.env.production?
+  #
+  # == Instantiating a new StringInquirer by wrapping a string inside it.
```

Comment:
@kaspth I have made the above changes as mentioned by you, and pushed the new commits.